benchmarks/benchmarks/spatial.py

```diff
@@ -34,15 +35,15 @@ def setup(self, mnr, cls_name):
         self.queries = np.concatenate((rng.standard_normal((r//2,m)),
                                        rng.standard_normal((r-r//2,m))+np.ones(m)))
 
-    def time_build(self, mnr, cls_name):
+    def time_build(self, mnr, cls_name, workers):
         """
         Constructing kd-tree
         =======================
         dim | # points |  time
         """
         m, n, r = mnr
         if cls_name == 'cKDTree_flat':
-            self.T = self.cls(self.data, leafsize=n)
+            self.T = self.cls(self.data, leafsize=n, workers=workers)
         else:
             self.cls(self.data)
```

Comment:
This doesn't benchmark `workers`. `cKDTree_flat` isn't one of the options, so I don't know why that branch is there and you've edited the wrong side.

Also note that benchmarks need to work with older versions of scipy so we can compare performance over time. So, in the setup you should detect if there is a `workers` argument, and only add it if there is.

```
try:
    self.cls(np.ones((1, 3)), workers=1)
    self.workers_arg = dict(workers=1)
except:
    self.workers_arg = {}
```

Reply:
Adressed in https://github.com/scipy/scipy/pull/14481/commits/d965a626aae7f2f90f4367abd753b3d52eb4061d .

Reply:
@peterbell10 can you please check this? CI (specifically Lint) passes now. I added a function to do the filtering. in these two places.